crates/rpc-types/src/genesis.rs

```diff
@@ -242,7 +242,8 @@ mod tests {
           "ecotoneTime": 0,
           "fjordTime": 0,
           "graniteTime": 0,
-          "holoceneTime": 0
+          "holoceneTime": 0,
```

Comment:
```suggestion
          "holoceneTime": 0,
          "isthmusTime": 0,
```
isthmus should be enabled if jovian is enabled